hooks/msghooks.py

```diff
@@ -120,6 +120,11 @@ def receive_message():
                 return f"{reply} Here you go: 🍪"
             else:
                 return f"{reply} No."
+        if re.match(r"sudo (pl(s|z|ease) )?make meh? a coo?kie", without_ping):
+            if not (message["user_id"] == 281362 or message["user_id"] == 497390 or message["user_id"] == 475866):
```

Comment:
Why are you targeting specific users? A simple random.choice or random() < 0.x would be more fun.

Reply:
ok (i am commiti g frk. phon )

Reply:
You still haven't removed that. Furthermore, I would expect sudo make me a cookie to work always for Vyxal ROs and never for others.

Reply:
Also, sudo probably doesn't need plszease after it.

Reply:
whats an RO

Reply:
RO = Room Owner